e2e/tests/LoggedIn/publish.e2e.spec.ts

```diff
@@ -659,7 +659,7 @@ const confirmCoAuthorInvitation = async (browser: Browser, user: Helpers.TestUse
     // clicking 'I am an author' link is blocked by cors
     const invitationLink = await page2
         .frameLocator('iframe')
-        .locator("a.button:has-text('I am an author')")
+        .locator("a:has-text('I am an author')")
```

Comment:
this has already been updated on the main branch, please always get the latest code from main